### ISSUE TITLE
background-position expected <bg-position> - false positive

### ISSUE THREAD

**Issue by djbriane** [CSSLint/csslint#257]
I've noticed that using single value for bg-position such as:

```css
background-position: center;
```

Causes the 'known-properties' rule to fire. According to MDN:

> bg-position
>
> Accepts one or two values, up to four values in supporting browsers. Negative <percentage> and <length> values are allowed.
> 
> If only one value is specified, the second value is assumed to be center.
> The first value represents the horizontal position and the second represents the vertical position (if at least one value is not a keyword).

https://developer.mozilla.org/en/CSS/background-position
http://www.w3.org/TR/css3-background/#background-position

Also seeing failures when using the legacy webkit gradient syntax (still needed for Android):
```css
background-image: -webkit-gradient(linear, left top, left bottom, from(rgb(255, 255, 255)),to(rgb(234, 235, 235)));
```

I couldn't see a flag that dealt with these issues directly - is this something that can change in the parser? Tried to do my homework but if I am missing something or this is a valid condition, please let me know! Thanks!

**Comment by frvge** [comment]
This should have been fixed in https://github.com/CSSLint/parser-lib/commit/0b2a4acdc3213aeee297742890e8689f7816bd5c . This will be released soon, so I'm marking this issue a duplicate of #514 .
